dynamic/Dockerfile

```diff
@@ -57,6 +57,10 @@ RUN microdnf install --nodocs hostname procps iptables iproute util-linux python
     && rm -rf /tmp/* \
     && microdnf clean all
 
+# Upgrade pip to 20.3.4 to get all the latest features
+# NOTE: 20.3.4 is the last version to support both Python 2.x & 3.x
+RUN pip install pip==20.3.4
```

Comment:
Did you verify the build? For me its not working, since we only have `pip3` installed and no `pip`:

Inside the container
```
[root@codepaper agent]# pip3 -V
pip 9.0.3 from /usr/lib/python3.6/site-packages (python 3.6)
[root@codepaper agent]# 
[root@codepaper agent]# pip -V
bash: pip: command not found
```

From the build:
```
------                                                                                                                                                                                                              
 > [stage-2  3/14] RUN pip install pip==20.3.4:                                                                                                                                                                     
#9 0.357 /bin/sh: pip: command not found                                                                                                                                                                            
------   
```

Furthermore, why do we need to upgrade `pip3` and cannot simply stick with the version that's coming with the `python3` package? Do we need Python 2 backwards compatibility for some special reason in our container?
And unless we really need to update, I'd much rather stick with the version coming with those packages, instead of manually having to update individual packages/libraries.

Reply:
@mdonkers We need Python2 support to be able to properly resolve all of the dependencies that the Python sensor needs to function.